doc/source/quickstart.rst

```diff
@@ -70,25 +40,49 @@ scripts and XML files to configure a case (see below) for the requested
 resolution, component set, and machine. **create_newcase** has several
 required arguments (invoke **create_newcase --help** for help).
 
-If running on a supported machine, (``$MACHINE``), that machine will
+The ``$PROJECT`` variable must be specified in either your 
+shell environment or as the ``--project`` argument to **create_newcase**.
+
+If running on a supported machine, that machine will
 normally be recognized automatically and therefore it is not required
 to specify the ``--machine`` argument to **create_newcase**, then
 invoke **create_newcase** as follows:
 
 .. code-block:: console
 
-    create_newcase --case $CASEROOT --compset $COMPSET --res $GRID 
+    ./create_newcase --case $CASEROOT --compset $COMPSET --res $GRID
+
+An example on NCAR machine cheyenne:
+
+.. code-block:: console
+
+    ./create_newcase --case /glade/scratch/[username]/cases/b.e20.B1850.f19_g17.test --compset B1850 --res f19_g17 
```

Comment:
I like that you give this example. For consistency, you should probably replace `[username]` with `$USER` (which in this case actually is a shell environment variable).

Reply:
fixed.